frontend/talentsearch/src/js/components/skills/SkillsInDetail/SkillsInDetail.tsx

```diff
@@ -104,7 +105,9 @@ const SkillsInDetail: React.FunctionComponent<SkillsInDetailProps> = ({
                   data-h2-font-style="b(underline)"
                   data-h2-display="b(flex)"
                   data-h2-align-items="b(center)"
-                  onClick={handleDelete}
+                  onClick={() => {
```

Comment:
Yeah, it was previously just acting directly on a function passed through props. I updated it to pass the ID up the tree with the function so the parent component knows the ID of the item being removed. Previously, the delete function passed in would need to figure the ID out itself.

With this new approach, this component its notifying the parent which skill is being deleted rather than the parent component needing to figure that out.

Reply:
Oh, it's passing the id now, of course.  Thanks!